index.bs

```diff
@@ -2365,48 +2365,99 @@ Reviews and Review Responsibilities</h4>
-	(for example through notices posted to <a href="https://lists.w3.org/Archives/Public/public-review-announce/">public-review-announce@w3.org</a>)
-	and were able to actually perform reviews of and provide comments on the specification.
-	A second objective is to encourage groups to request reviews
-	early enough that comments and suggested changes
-	can still be reasonably incorporated in response to the review.
-	Before approving transitions,
-	the [=Director=] will consider who has been explicitly offered
-	a reasonable opportunity to review the document,
-	who has provided comments,
-	the record of requests to and responses from reviewers,
-	especially <a href="https://www.w3.org/Guide/process/charter.html#horizontal-review">W3C Horizontal Groups</a> [[CHARTER]]
-	and groups identified as dependencies in the charter
-	or identified as <a href="https://www.w3.org/2001/11/StdLiaison.html">liaisons</a> [[LIAISON]],
-	and seek evidence of clear communication to the general public
-	about appropriate times and which content to review
-	and whether such reviews actually occurred.
+	The objective of <dfn id=dfn-wide-review export>wide review</dfn> is to seek
+	feedback on a specification from the entire web community, including the general
+	public. The goal is to make the specification as robust as possible before it
+	becomes a [=W3C Recommendation=]. 
 
-	For example,
-	inviting review of new or significantly revised sections published in Working Drafts,
-	and tracking those comments
-	and the [=Working Group=]'s responses,
-	is generally a good practice which would often be considered positive evidence of wide review.
-	[=Working Groups=] <em class="rfc2119">should</em> announce to other W3C Working Groups
-	as well as the general public,
-	especially those affected by this specification,
-	a proposal to enter [=Candidate Recommendation=]
-	(for example in approximately 28 days).
-	By contrast a generic statement in a document
-	requesting review at any time
-	is likely not to be considered as sufficient evidence
-	that the group has solicited wide review.
-
-	A [=Working Group=] could present evidence that wide review has been received,
-	irrespective of solicitation.
-	But it is important to note that receiving many detailed reviews
-	is not necessarily the same as wide review,
-	since they might only represent comment
-	from a small segment of the relevant stakeholder community.
+	Wide review <em class="rfc2119">should</em> be sought continuously as a
+	specification matures, to allow different parts of the web community to iterate
+	on requirements, problems, and solutions. 
+
+	When a [=First Public Working Draft=] of a specification is published, a notification
+	<em class="rfc2119">must</em> be sent to <a href="https://lists.w3.org/Archives/Public/public-review-announce/">public-review-announce@w3.org</a>. 
+
+	As a specification progresses along the Recommendation Track, Working Groups
```

Comment:
Is there a reason you worry that groups could be offered different turnaround times at this particular point, as opposed to any other?

I'm hesitant to say much about wat happens before FPWD because more often than not these days that happens in a CG and outside of the Process' remit. It's tricky though.

Reply:
I do know that several HR groups have expressed a desire to participate in reviews in those early stages, though - e.g. the Privacy Interest Group wants to (and does) review WICG incubations.

Reply:
"As a specification is being prepared for, and then moved along, the Recommendation track…" ?

Reply:
> Is there a reason you worry that groups could be offered different turnaround times at this particular point, as opposed to any other?

I think if it looks like W3C is treating different groups differently without some good reason then it would reflect badly on W3C. 

I'm only picking it up here because this is the change being proposed. I suppose I could try to review the whole Process for it and open a separate issue, but I wasn't particularly aware of it as a problem elsewhere before.

Reply:
I've updated the text to use @dwsinger's suggestion:
>As a specification is being prepared for, and moved along, the Recommendation Track, the group responsible for the specification must solicit a horizontal review from each of the horizontal review groups (in many cases, submitting a self-review to the reviewing group will satisfy this requirement).>

Reply:
This new wording works for me.